src/api/app/views/webui/project/show_actions/_patchinfo.html.haml

```diff
@@ -1,7 +1,5 @@
 - unless has_patchinfo
   %li.nav-item
     = link_to(patchinfo_path(project: project), method: :post, class: 'nav-link', title: 'Create Patchinfo') do
-      %span.fa-stack
-        %i.fas.fa-band-aid.fa-lg.fa-stack-1x
-        %i.fas.fa-plus-circle.fa-stack-1x.top-icon
```

Comment:
This doesn't only change `fa-lg` to `fa-fw`. Could you elaborate why the icon stack was removed?

Reply:
I've removed this icon stack and changed it to a "normal" icon.

Reply:
Maybe I was not clear enough when I asked before... why exactly?